metadata.yaml

```diff
@@ -46,6 +46,8 @@ containers:
     mounts:
       - storage: mongodb
         location: /var/lib/mongodb
+      - storage: mongodb-logs
+        location: /var/log/mongodb
```

Comment:
I think (please check) adding storage will break "juju refresh".
Could we add logs to the data dir right now? and decide if it should be separate dir next few weeks

Reply:
sure, let's add logs for data dir first and then adjust when necessary. Same needs to be done for VM charm anyway